src/driver/latin_hypercube_random_test_point_generator.py

```diff
@@ -36,16 +36,23 @@
 class LatinHypercubeRandomTestPointGenerator(TestPointGenerator):
     """Generates a collection of test points for a given simulation."""
 
-    def __init__(self, dataset: DataSet, rng: np.random.Generator):
+    def __init__(
+        self,
+        dataset: DataSet,
+        rng: np.random.Generator,
+        npoints: int = MINIMUM_NUMBER_OF_TEST_POINTS,
```

Comment:
Per our previous discussion, it is often desirable to choose `npoints` based on `self._npublishers`. 

How about (1) keep it as it is in your PR (2) I will refactor this class in the following logic.

Add an arg called `npoints_generator`, for example,
```
npoints_generator = lambda num_pubs: min(MAX_NUM_TEST_POINTS, 100 * num_pubs)
```
And in `__init__(self, dataset, rng, npoints_generator, npoints=None)`,
```
if npoints is None:
    self._npoints = npoints_generator(self._npublishers)
else:
    self._npoints = npoints
```